recipes-bsp/bootfiles/rpi-config_git.bb

```diff
@@ -178,6 +178,13 @@ do_deploy() {
         echo "dtoverlay=${VC4DTBO}" >> ${DEPLOYDIR}/${BOOTFILES_DIR_NAME}/config.txt
     fi
 
+    # Choose Camera Sensor to be used, default imx219 sensor
+    if [ "${RASPBERRYPI_CAMERA_V2}" = "1" ]; then
```

Comment:
Please document this variable in the docs. See other examples there.

Reply:
Ok, thanks.